diff -Naur vendor.orig/bundle/ruby/2.5.0/gems/nanoc-4.9.9/lib/nanoc/filters/colorize_syntax.rb vendor/bundle/ruby/2.5.0/gems/nanoc-4.9.9/lib/nanoc/filters/colorize_syntax.rb
--- vendor.orig/bundle/ruby/2.5.0/gems/nanoc-4.9.9/lib/nanoc/filters/colorize_syntax.rb	2018-09-13 01:23:19.096066864 +0900
+++ vendor/bundle/ruby/2.5.0/gems/nanoc-4.9.9/lib/nanoc/filters/colorize_syntax.rb	2018-09-13 01:28:32.179975348 +0900
@@ -21,8 +21,8 @@
 
       # Colorize
       doc = parse(content, parser, params.fetch(:is_fullpage, false))
-      selector = params[:outside_pre] ? 'code' : 'pre > code'
-      doc.css(selector).each do |element|
+      selector = params[:outside_pre] ? './/*[local-name()="code"]' : './/*[local-name()="pre"]/*[local-name()="code"]'
+      doc.xpath(selector).each do |element|
         # Get language
         extracted_language = extract_language(element)
 
